### PR TITLE
[dagster-powerbi] Add basic docs

### PR DESCRIPTION
**Pull request by benpankow**
## Summary

Introduce initial docs for Power BI, to send to initial design partners. These are hidden from nav for now since the integration is not being widely publicized.

### CODE REVIEW

**Comment by benpankow** [comment]
* **#24269** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24269?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @benpankow and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: examples/docs_snippets/docs_snippets/integrations/power-bi/customize-power-bi-asset-defs.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    credentials=PowerBIServicePrincipal(
+        client_id=EnvVar("POWER_BI_CLIENT_ID"),
+        client_secret=EnvVar("POWER_BI_CLIENT_SECRET"),
+        tenant_id=EnvVar("POWER_BI_TENANT_ID"),
+    ),
+    workspace_id=EnvVar("POWER_BI_WORKSPACE_ID"),
+)
+
+
+class MyCustomPowerBITranslator(DagsterPowerBITranslator):
```
Comment (erinkcochran87) [inline examples/docs_snippets/docs_snippets/integrations/power-bi/customize-power-bi-asset-defs.py:25]: Perhaps add some comments in here to highlight and explain what's happening?

File: examples/docs_snippets/docs_snippets/integrations/power-bi/multiple-power-bi-workspaces.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    credentials=credentials,
+    workspace_id="726c94ff-c408-4f43-8edf-61fbfa1753c7",
+)
+
+marketing_team_workspace = PowerBIWorkspace(
+    credentials=credentials,
+    workspace_id="8b7f815d-4e64-40dd-993c-cfa4fb12edee",
+)
+
+defs = Definitions.merge(
```
Comment (erinkcochran87) [inline examples/docs_snippets/docs_snippets/integrations/power-bi/multiple-power-bi-workspaces.py:25]: Perhaps add a comment here to highlight this?

File: docs/content/integrations/powerbi.mdx [hunk 3]
```diff
@@ -0,0 +1,179 @@
+---
+title: "Dagster & Power BI"
+description: "Represent Power BI semantic models, data sources, reports, and dashboards as assets in Dagster."
+---
+
+# Dagster & Power BI
+
+<ExperimentalCallout />
+
+This guide provides instructions for using Dagster with Power BI.
```
Comment (erinkcochran87) [inline docs/content/integrations/powerbi.mdx:10]: Do you plan to flesh these out more in the near future? If not, consider adding a sentence or two here about why using Power BI with Dagster is beneficial.

File: docs/content/integrations/powerbi.mdx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    workspace_id=EnvVar("POWER_BI_WORKSPACE_ID"),
+)
+defs = resource.build_defs(enable_refresh_semantic_models=True)
+```
+
+You can then add these semantic models to jobs. They can also be targets of Dagster sensors or schedules, allowing you to trigger model refreshes on a regular cadence or based on specific conditions.
+
+---
+
+## Loading Power BI assets from multiple workspaces
```
Comment (erinkcochran87) [inline docs/content/integrations/powerbi.mdx:146]: Should this be part of the first section? Ex:

```
## Representing Power BI assets in Dagster

...

### Loading Power BI assets from multiple workspaces

...
```

**Review by erinkcochran87 — COMMENTED** [review]
I made some updates to bring this in line with the current site's styling, just so we didn't have to go back and forth on it.

I left a few comments, but they're minor. Also, should this be accessible at this point? Is the page not being included in the sidenav or `integrations.md` intentional?

File: docs/content/integrations/powerbi.mdx (hunk 3 above)
Comment (benpankow) [reply docs/content/integrations/powerbi.mdx:9]: Added a few sentences, but plan to revisit this later on!

**Comment by benpankow** [comment]
> should this be accessible at this point? Is the page not being included in the sidenav or integrations.md intentional?

Right now this is just to send to design partners, the integration isn't widely publicized

File: docs/content/integrations/powerbi.mdx [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+# Using Power BI
+
+<ExperimentalCallout />
+
+This guide provides instructions for using Dagster with Power BI. Your Power BI assets, such as semantic models, data sources, reports, and dashboards, can be represented in the Dagster asset graph, allowing you to track lineage and dependencies between Power BI assets and upstream data assets you are already modeling in Dagster. You can also use Dagster to orchestrate Power BI semantic models, allowing you to trigger refreshes of these models on a cadence or based on upstream data changes.
+
+## What you'll learn
+
+- How to represent Power BI assets in the Dagster asset graph, including lineage to other Dagster assets.
+
```
Comment (sryza) [inline docs/content/integrations/powerbi.mdx:14]: Do we need newlines between these?

File: docs/content/integrations/powerbi.mdx [hunk 6]
```diff
@@ -0,0 +1,172 @@
+---
+title: "Using Power BI"
```
Comment (sryza) [inline docs/content/integrations/powerbi.mdx:2]: We normally include a description in the header, right?

File: docs/content/integrations/powerbi.mdx [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+---
+
+# Using Power BI
```
Comment (sryza) [inline docs/content/integrations/powerbi.mdx:5]: Using Dagster with Power BI?

File: docs/vale/styles/config/vocabularies/Dagster/accept.txt [hunk 8]
```diff
@@ -132,3 +133,15 @@ uncomment
 unpartitioned
 vCPU
 vCPUs
+we have
```
Comment (sryza) [inline docs/vale/styles/config/vocabularies/Dagster/accept.txt:136]: we have?

**Review by sryza — APPROVED** [review]
A few small comments, otherwise LGTM

**Pull request merged**